### ISSUE TITLE
Forum posts are out of order

### ISSUE THREAD

**Issue by Data66PF** [moxi9/phpfox#54]
In the pf community you can see it best, http://community.phpfox.com/forum/thread/51971/plz-help-me-how-to-install-video-app-for-phpfox-v4/ just one example. The posts are out of order.

It goes from first to last to next to last to I don't know. 

Ok I see what it did. It reversed itself. It is not showing oldest down to newest. It needs to in order to follow the posts. It is showing Main post, then newest down to oldest but that makes the conversation hard to follow until you notice what happened.

In that thread, it even renumbered the posts. Post # 2 is actually Post # 12

**Comment by RaymondBenc** [comment]
Yeah, sorta major change. Another push will be added so users can chose. We will default it back to normal.
